docs/changehistory/NextVersion.md

```diff
@@ -61,26 +61,29 @@ For detailed information about the recommended KindOfQuantity to use in your too
 
 ## Incremental Schema Loading
 
-Incremental Schema Loading support has been added to the the backend on `IModelDb` and `IModelApp` on frontends. In this release, the incremental schema loading is disabled by default, but clients can enable it using the incrementalSchemaLoading setting on the `IModelDb` and `IModelApp` options. Incremental schema loading allows to load a schema partially. Clients can recieve their schema and its elemehts while references and lazy loaded schema elements are loaded in the background.
-
-### Examples
-*Frontend*
-```
-await IModelApp.startup({
-  incrementalSchemaLoading: "enabled"
-});
-
-await IModelApp.startup({
-  incrementalSchemaLoading: "disabled"
-});
-```
-*Backend*
-```
-await IModelHost.startup({
-  incrementalSchemaLoading: "enabled",
-});
-
-await IModelHost.startup({
-  incrementalSchemaLoading: "disabled",
-});
-```
\ No newline at end of file
+*Incremental Schema Loading* support has been added to the backend on `IModelDb` and `IModelConnection` on frontends.
+
+Clients can control how the schemas are loaded through their application context, using the `incrementalSchemaLoading` option when initializing their `IModelHost` or `IModelApp`. By default, incremental schema loading is disabled, so clients have to enable this feature if they want to use it.
+
+Incremental schema loading allows to load a schema partially. Clients can receive their schema and its elements while references and lazy loaded schema elements are loaded in the background.
+
+- Frontend examples:
+  ```ts
+  await IModelApp.startup({
+    incrementalSchemaLoading: "enabled"
+  });
```

Comment:
Nitpick: could you place `startup` calls in separate code blocks ?

Now it looks like that you can call `startup` one after another and the second call will disable incremental loading.